pages/common/hexyl.md

```diff
@@ -0,0 +1,20 @@
+# hexyl
+
+> A simple hex viewer for the terminal. Uses colored output to distinguish different categories of bytes.
+> More information: <https://github.com/sharkdp/hexyl>.
+
+- Print the hexadecimal representation of a file:
+
+`hexyl {{file}}`
+
+- Print the hexadecimal representation of a file, but interpret only n bytes of the input:
```

Comment:
Perhaps this will read better?
```suggestion
- Print the hexadecimal representation of the first n bytes of a file:
```

Also, could `file` be `path/to/file` please? Same goes for the other examples.